### PR TITLE
Add MAINTAINERS file

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Alex Somesan @alexsomesan <alex.somesan@coreos.com>
+Carlos Panato @cpanato <carlos.panato@coreos.com>
+Ed Rooth @sym3tri <ed.rooth@coreos.com>
+Erik Nobel @erikkn <erik.nobel@coreos.com>
+Karen Almog @kalmog <karen.almog@coreos.com>
+Lucas Servén @squat <lucas.serven@coreos.com>
+Max Inden @mxinden <max.inden@coreos.com>
+Sergiusz Urbaniak @s-urbaniak <sergiusz.urbaniak@coreos.com>
+
```
Comment (squat) [inline MAINTAINERS]: Looks like an extra new-line

Comment (mxinden) [reply MAINTAINERS]: @squat Just following Unix convention. I don't have a strong opinion on this. Shall I remove it?

Comment (squat) [reply MAINTAINERS]: this is distinct from unix convention. This is an EXTRA newline apart from the trailing newline after the file.

Comment (squat) [reply MAINTAINERS]: When the file does not have a trailing newline, GitHub renders a red :no_entry_sign:

Comment (mxinden) [reply MAINTAINERS]: Ah, sorry for the confusion. Should be fixed now.

Comment (squat) [reply MAINTAINERS]: yes looks good now. This has the correct newline at the end of the file.

**Pull request merged**